src/components/PopupWithProfilePicForm.js

```diff
@@ -0,0 +1,33 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithProfilePicForm extends Popup {
```

Comment:
Needs correction: you don't need a special class for the avatar form, use PopupWithForm